contracts/crowdsale/distribution/RefundableCrowdsale.sol

```diff
@@ -3,31 +3,31 @@ pragma solidity ^0.4.24;
 
 import "../../math/SafeMath.sol";
 import "./FinalizableCrowdsale.sol";
-import "./utils/RefundVault.sol";
+import "../../payment/RefundEscrow.sol";
```

Comment:
I think `RefundEscrow` is very much crowdsale-specific so it should be with the crowdsale stuff. Potentially in this same file. Thoughts?

Reply:
I disagree: a `RefundEscrow` is a _payment_ method, a crowdsale typically using it doesn't make it crowdsale-specific. As an example, platforms like Amazon or eBay also offer refunds and act as an escrow themselves, and could extend their payment systems to add support for multiple payers.